src/main/resources/db/changelog/changelog.sql

```diff
@@ -1040,3 +1040,19 @@ ADD INDEX `TimelineMetadata-SessionInstanceGuid` (sessionInstanceGuid);
 
 ALTER TABLE `DemographicsValues`
 ADD COLUMN `invalidity` varchar(512) DEFAULT NULL;
+
+-- changeset bridge:76
+
+CREATE TABLE IF NOT EXISTS `Alerts` (
+    `id` varchar(60) NOT NULL,
+    `createdOn` bigint(20) NOT NULL,
+    `studyId` varchar(60) NOT NULL,
+    `appId` varchar(60) NOT NULL,
+    `userId` varchar(255) NOT NULL,
+    `category` varchar(255) NOT NULL,
+    `data` varchar(2048) NOT NULL,
+    PRIMARY KEY (`id`),
+    UNIQUE KEY (`studyId`, `appId`, `userId`, `category`),
```

Comment:
Does this create an index on app and study? If not, we need to add this index.

Same with index on app+userId and index on app+study+userId.